documentation/modules/mtv-aio-buffer.adoc

```diff
@@ -0,0 +1,246 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: CONCEPT
+[id="mtv-aio-buffer_{context}"]
+= AIO buffer counts for customer usage
+
+This document is intended to summarize the performance team's testing on the impact of changes to the AIO buffer count, specifically to assess any effects on datastore read rates and overall migration time when using the {project-first} product.
+
+.Additional information
+
+* link:https://docs.vmware.com/en/VMware-vSphere/7.0/vsphere-vddk-programming-guide/GUID-5D166ED1-7205-4110-8D72-0C51BB63CC3D.html[Best Practices for NBD Transport]
+* link:https://forums.veeam.com/vmware-vsphere-f24/solution-for-poor-nbd-performance-t93084.html[Solution for poor NBD performance]
+
+[id="mtv-aio-performance-results_{context}"]
+== Performance Testing Results
+
+There were several extensive tests, which were conducted across multiple configurations:
+
+* Single VMware ESXi server compared to multiple ESXi servers.
+* Single virtual machine (VM) compared to multiple VMs.
+
+[id="mtv-aio-buffer-key-findings_{context}"]
+=== Key findings
+
+* The best migration performance was achieved by migrating using multiple VMs (10) on a single ESXi host with the following values:
+** `VixDiskLib.nfcAio.Session.BufSizeIn64KB=16`
+** `vixDiskLib.nfcAio.Session.BufCount=4`
+
+* The following improvements were noted when using AIO buffer (Asynchronous Buffer Counts) settings:
+** Migration time was reduced by *31.1%*, from `0:24:32` to `0:16:54`.
+** Read rate was increased from `347.83MB/s` to `504.93MB/s`.
+
+* There was no significant improvement observed when using AIO buffer settings with a single VM.
+
+* There was no significant improvement observed when using AIO buffer settings with multiple VMs from multiple hosts.
+
+[id="mtv-enable-aio-buffer_{context}"]
+== Enabling AIO buffer configuration
+
+.Validating Controller Pod support for AIO values
+
+* Ensure that the `forklift-controller` pod in the `openshift-mtv` namespace supports the AIO buffer values. Check the environment variables of the pod by running:
+
++
+[source,terminal]
+----
+oc get pod forklift-controller-5b45f66949-vgzhx -n openshift-mtv -o yaml
+----
++
+
+* Check for the following lines in the output:
++
+[source,termina]
+----
+...
+\- name: VIRT\_V2V\_EXTRA\_ARGS
+\- name: VIRT\_V2V\_EXTRA\_CONF\_CONFIG\_MAP
+...
+----
+
+.Editing ForkliftController Configuration
+
+* In the `openshift-mtv namespace`, edit the `ForkliftController` object to include the AIO buffer values.
++
+Add the following under the spec section:
++
+[source,termina]
+----
+oc edit forkliftcontroller -n openshift-mtv`
+----
+
+* For example, to add the following values:
+** virt_v2v_extra_args:
++
+[source,termina]
+----
+--vddk-config /mnt/extra-v2v-conf/input.conf
+----
++
+[source,termina]
+----
+virt_v2v_extra_conf_config_map: "`perf`"
```

Comment:
@anarnold97 
please change the following line with this one : 
virt_v2v_extra_conf_config_map: "perf"